pkg/ovs/openflow/ofctrl_packetin.go

```diff
@@ -20,6 +20,14 @@ import (
 	"github.com/contiv/ofnet/ofctrl"
 )
 
+const (
+	NPPacketInMeterId = 1
+	TFPacketInMeterID = 2
+
+	NPPacketInRate = 100
+	TFPacketInRate = 200
```

Comment:
I am not sure why these constants are defined here and not in `pkg/agent/openflow/client.go` where we use them?

Also, to match other constants, I think they should be named PacketInMeterIdNP, PacketInMeterIdTF, etc.

Reply:
Yeah I can move them to `client.go`. 
The reason I put it here is that we use meter entries for rate limiting on packet-in events, these meter-related constants seem to have a closer relationship with `ofctrl_packetin.go` ? 
Will change names of these constants.

Reply:
Then `pkg/agent/openflow/packetin.go` may be a good compromise (with the reason constants, etc.), but IMO these belong in the `pkg/agent/openflow` package and definitely not under `pkg/ovs`

Reply:
Got it.